R/records.R

```diff
@@ -73,12 +74,16 @@ recordExists <- function(formId, recordId) {
 #' ))
 #'
 #' }
-addRecord <- function(formId, parentRecordId = NA_character_, fieldValues) {
+addRecord <- function(formId, parentRecordId = NA_character_, fieldValues, recordId = NA_character_) {
   stopifnot(is.character(formId))
   stopifnot(is.character(parentRecordId))
   stopifnot(is.list(fieldValues))
-  
-  recordId <- cuid()
+  stopifnot(is.character(recordId))
```

Comment:
This does not check whether a record with this id already exists, if provided. We had previously fixed this so that users would get feedback if they tried to update a record that didn't exist, they would get an error. We don't support this through the API, so you need to do a check for the existence of the record with getRecord()